@uportal/dashboard-carousel/vue.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  // options...
+  runtimeCompiler: true,
```

Comment:
Does the dashboard carousel need the vue template compiler included in the runtime?
I thought the new renderer projects are handling templating?

Reply:
good catch, its copy pasta from copying the proxy config